src/jdk.incubator.vector/share/classes/jdk/incubator/vector/IntVector.java

```diff
@@ -1780,6 +1787,25 @@ IntVector abs() {
         return lanewise(ABS);
     }
 
+    // Popcount
+    /**
+     * Computes the number of one-bits in the two's complement
+     * binary representation of the specified value
+     *
+     * This method is also equivalent to the expression
+     * {@link #lanewise(VectorOperators.Unary)
+     *  lanewise}{@code (}{@link VectorOperators#POPCNT
+     * }{@code )}.
+     *
+     * @return number of one-bits
+     * @see VectorOperators#POPCNT
+     * @see #lanewise(VectorOperators.Unary)
+     */
+    @ForceInline
+    public final
+    IntVector popcnt() {
```

Comment:
I recommend not adding such a specific lanewise method for now, and let users use the operation directly.